google/gax/retry.py

```diff
@@ -120,10 +120,11 @@ def inner(*args):
                 return to_call(*args)
             except Exception as exception:  # pylint: disable=broad-except
                 code = config.exc_to_code(exception)
+
+                # Do not be greedy; if this is an exception that we do
```

Comment:
nit: I'm not sure "do not be greedy" is going to make sense to someone reading this code without knowledge of its history.